### PR TITLE
Add com.github.iwalton3.jellyfin-mpv-shim.

### PR DESCRIPTION
**Pull request by iwalton3**
[Jellyfin MPV Shim](https://github.com/iwalton3/jellyfin-mpv-shim) is a desktop media player client for the [Jellyfin](https://jellyfin.org/) Media Server.

https://github.com/iwalton3/jellyfin-mpv-shim/issues/47

### CODE REVIEW

**Comment by iwalton3** [comment]
bot, build com.github.iwalton3.jellyfin-mpv-shim

**Comment by iwalton3** [comment]
bot, build com.github.iwalton3.jellyfin-mpv-shim

**Comment by iwalton3** [comment]
It looks like this test build has been stuck in the queue for 3 hours. Is this normal?

**Comment by iwalton3** [comment]
bot, build com.github.iwalton3.jellyfin-mpv-shim

**Comment by iwalton3** [comment]
bot, build com.github.iwalton3.jellyfin-mpv-shim

**Comment by iwalton3** [comment]
bot, build com.github.iwalton3.jellyfin-mpv-shim

File: com.github.iwalton3.jellyfin-mpv-shim.json [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    "--socket=wayland",
+    "--device=all",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-pictures",
+    "--talk-name=org.gtk.vfs",
+    "--talk-name=org.gtk.vfs.*",
+    "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
+    "--env=LC_NUMERIC=C",
+    "--persist=.config/jellyfin-mpv-shim"
```
Comment (barthalion) [inline com.github.iwalton3.jellyfin-mpv-shim.json:19]: This is not needed. Your application should respect $XDG_CONFIG_HOME and put config files there.

File: com.github.iwalton3.jellyfin-mpv-shim.json [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+              "type": "archive",
+              "url": "https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.6.tar.xz",
+              "sha256": "8351328cdfbcb2432e63938721dd781eb8c11ebc56e3a89d0f84576b96002c61"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "integration",
```
Comment (barthalion) [inline com.github.iwalton3.jellyfin-mpv-shim.json:229]: Ideally all these files should be maintained in your repository and shipped as part of the tarball, although I'm not sure how feasible is it with pip.

**Comment by iwalton3** [comment]
bot, build com.github.iwalton3.jellyfin-mpv-shim

File: com.github.iwalton3.jellyfin-mpv-shim.json (hunk 2 above)
Comment (iwalton3) [reply com.github.iwalton3.jellyfin-mpv-shim.json]: These files have been moved to the pip distribution.

File: com.github.iwalton3.jellyfin-mpv-shim.json (hunk 1 above)
Comment (iwalton3) [reply com.github.iwalton3.jellyfin-mpv-shim.json]: The persist rule has been removed.

**Comment by iwalton3** [comment]
bot, build com.github.iwalton3.jellyfin-mpv-shim

**Pull request closed without merging**